fdbserver/TenantBalancer.actor.cpp

```diff
@@ -938,12 +951,12 @@ ACTOR Future<Void> getActiveMovements(TenantBalancer* self, GetActiveMovementsRe
 	++self->getActiveMovementsRequests;
 
 	TraceEvent(SevDebug, "TenantBalancerGetActiveMovements", self->tbi.id())
-	    .detail("PrefixFilter", req.prefixFilter)
+	    .detail("PrefixFilter", req.prefixFilter.present() ? req.prefixFilter.get().toString() : "[not set]")
 	    .detail("LocationFilter",
 	            req.locationFilter.present()
 	                ? TenantBalancerInterface::movementLocationToString(req.locationFilter.get())
 	                : "[not set]")
-	    .detail("PeerClusterFilter", req.peerDatabaseConnectionStringFilter);
+	    .detail("PeerClusterFilter", req.peerDatabaseConnectionStringFilter.orDefault("[not set]"));
```

Comment:
You don't need this, logging an optional does this automatically.

```suggestion
	    .detail("PeerClusterFilter", req.peerDatabaseConnectionStringFilter);
```